client/src/screens/settings.screen.js

```diff
@@ -137,48 +153,79 @@ class Settings extends Component {
     this.props.dispatch(logout());
   }
 
-  render() {
-    const { loading, user } = this.props;
+  updateDisplayName = (value) => {
+    this.setState({ promptVisible: false });
+    this.props
+      .updateUserProfile({ displayName: value });
+  }
+
+  openPrompt = () => {
+    this.setState({ promptVisible: true });
+  };
+
+    closePrompt = () => {
```

Comment:
is it necessary to have separate functions for close vs cancel?

Reply:
in future, maybe? I can collapse if we dont care

Reply:
at least fix indentation plz